packages/react-app/theme.js

```diff
@@ -30,6 +30,43 @@ const theme = extendTheme({
       accentGreen: "#B5DC3A",
       baseGreen: "#EAFFA9",
     },
+    // color schemes
+    customBaseColorScheme: {
+      // hover bg (light)
+      50: "#FFFCFC",
+      100: "#cfd2d9",
+      // color (dark)
+      // hover bg (dark) with opacity
+      200: "#FFFCFC",
+      300: "#9da3b2",
+      400: "#848ca0",
+      500: "#182232",
+      // color (light)
+      600: "#182232",
+      700: "#131b27",
+      800: "#101722",
+      900: "#0c1017",
+    },
```

Comment:
Nicee !! Just curious how did you know which number corresponds to what result in the button,  example  600 => text color (light) 

I think the comments mapping only true when you use `variant="outline"`

Because if you `variant="solid"` then, 600 != text-color 

Just cursion, was thinking of configuring for our blue buttons on figma

Reply:
I just checked the chrome dev tool and see which variable was applied (--var--....--600) or whatever.

> I think the comments mapping only true when you use variant="outline"

Only tested with this, yes! But maybe as we create other buttons we can see the corresponding colors and see if they make sense.

(I generated the initial pallete with ChatGPT haha)

Reply:
Ohh yup, makes sense ! Thanks 🙌